pkg/construct/construct.go

```diff
@@ -29,7 +29,7 @@ import (
 
 var log = logging.Logger("service")
 var providerIndexNamespace = datastore.NewKey("providerindex/")
-var providerIndexPublisherNamespace = providerIndexNamespace.ChildString("publisher/")
+var providerIndexPublisherNamespace = providerIndexNamespace.Child(datastore.NewKey("publisher/"))
```

Comment:
Why did change?

Reply:
childstring is a bit error prone on slashes